.github/workflows/cli-deploy.yml

```
@@ -0,0 +1,44 @@
+name: "CLI deploy"
+on:
+  workflow_dispatch:
+    branches:
```

actionlint (1.7.4)

expected "inputs" key for "workflow_dispatch" section but got "branches"

(syntax-check)